### PR TITLE
Fixed ruby_decorators not being available

### PR DESCRIPTION
**Pull request by philsturgeon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,7 +148,7 @@ Easy! Check your logs for `user_agent=service-name; app_name=service-name;` The
 
 - **Ruby:** v2.2 - v2.4
 - **Rails:** v4.2 - v5.1
-- **Faraday:** v0.8 - v0.13
```
Comment (philsturgeon) [inline README.md]: Not part of this PR, just a lie I noticed in the README

File: lib/we/call/annotations.rb [hunk 2]
```diff
@@ -2,6 +2,7 @@ module We
   module Call
     module Annotations
       # Enable annotations
+      require "ruby_decorators"
```
Comment (bagmangood) [inline lib/we/call/annotations.rb]: shouldn't this be at the top of the file?
